### ISSUE TITLE
Could not load a Sketch preview "images/853732577995ec08625706620b0235b0184b90e8.pdf"

### ISSUE THREAD

**Issue by AnnMilesCHR** [xlayers/xlayers#321]
1. Go to 'Upload A Design from Your Computer '
2. Select a '.sketch' file and click 'Open'
3. See error

Sketch v56.2 (81672)
iMac running macOS Mojave 10.14.6
Chrome v76.0.3809.100 (Official Build) (64-bit)

**Comment by Jefiozie** [comment]
Thank you for submitting this issue. Is it possible to share the sketch file you uploaded? This way we can assist you better why reproducing the issue. 

Thanks

**Comment by AnnMilesCHR** [comment]
Ann Miles has shared a OneDrive for Business file with you. To view it, click the link below.
<https://chrsolutionsinc649-my.sharepoint.com/:u:/g/personal/ann_miles_chrsolutions_com/Ee48F6DzOjhMgbHwIwl2dBkBYZUSM1_zj7KPJJgzjEO_lw>
[https://r1.res.office365.com/owa/prem/images/dc-generic_20.png]<https://chrsolutionsinc649-my.sharepoint.com/:u:/g/personal/ann_miles_chrsolutions_com/Ee48F6DzOjhMgbHwIwl2dBkBYZUSM1_zj7KPJJgzjEO_lw>
Admin.sketch<https://chrsolutionsinc649-my.sharepoint.com/:u:/g/personal/ann_miles_chrsolutions_com/Ee48F6DzOjhMgbHwIwl2dBkBYZUSM1_zj7KPJJgzjEO_lw>

I actually had this problem with every .sketch file I tried use both the drag n drop or browser options. Here's one of the files I tied. I'm guessing they're just too large?



Ann Miles
Lead UX Designer/Developer

CHR Solutions, Inc.

ann.miles@chrsolutions.com

www.chrsolutions.com

________________________________

**Comment by manekinekko** [comment]
I have investigated this issue and here is a fix for this bug:

### tl;dr
We need to move the PDF check section to the top of the if/else block:
https://github.com/xlayers/xlayers/blob/ce8c800d58252db216b78e2a285729fc73939575/src/app/core/sketch.service.ts#L154-L157


### Details
This sketch file contains a PDF file under `images/xxxx.pdf`. So our parser treats this as an image because that was the case for the previous version of SketchApp. 

The we tries to load that file assuming it's a valid image:
https://github.com/xlayers/xlayers/blob/ce8c800d58252db216b78e2a285729fc73939575/src/app/core/sketch.service.ts#L96

And because it's not a valid image file. It fails.

FYI, we have decided not to parse PDF files for now and because of https://github.com/xlayers/xlayers/issues/200
